### ISSUE TITLE
Is this really free ban ?

### ISSUE THREAD

**Issue by Reraz** [hydy100/R3nzSkin#177]
I am an old user of R3nzskin and i wonder if with this skin changer there will be no problem like the old one.

**Comment by Ashantifragile** [comment]
nope, r3nz skin doesn't charge you and it's completely free. so it's probably hoax. i can vouch for this skin changer because i've been using this for many months now so yeah
